content/en/docs/Contribute/_index.md

```diff
@@ -0,0 +1,528 @@
+direction TB
+1[1. Go to opentelemetry.io repository] --> 2[2. Select New Pull Request]
+2 --> 3[3. Select compare across forks]
+3 --> 4[4. Select your fork from<br>head repository drop-down menu]
+end
+subgraph second [ ]
+direction TB
+5[5. Select your branch from<br>the compare drop-down menu] --> 6[6. Select Create Pull Request]
+6 --> 7[7. Add a description<br>to your PR]
+7 --> 8[8. Select Create pull request]
+end
+
+first --> second
+
+classDef grey fill:#dddddd,stroke:#ffffff,stroke-width:px,color:#000000, font-size:15px;
+classDef white fill:#ffffff,stroke:#000,stroke-width:px,color:#000,font-weight:bold
+class 1,2,3,4,5,6,7,8 grey
+class first,second white
+```
+
+Figure 3. Steps to open a PR from your fork to the
+[opentelemetry.io](https://github.com/open-telemetry/opentelemetry.io).
+
+1. In a web browser, go to the
+   [`opentelemetry.io`](https://github.com/open-telemetry/opentelemetry.io)
+   repository.
+1. Select **New Pull Request**.
+1. Select **compare across forks**.
+1. From the **head repository** drop-down menu, select your fork.
+1. From the **compare** drop-down menu, select your branch.
+1. Select **Create Pull Request**.
+1. Add a description for your pull request:
+
+   - **Title** (50 characters or less): Summarize the intent of the change.
+   - **Description**: Describe the change in more detail.
+
+     - If there is a related GitHub issue, include `Fixes #12345` or
+       `Closes #12345` in the description. GitHub's automation closes the
+       mentioned issue after merging the PR if used. If there are other related
+       PRs, link those as well.
+     - If you want advice on something specific, include any questions you'd
+       like reviewers to think about in your description.
+
+1. Select the **Create pull request** button.
+
+Your pull request is available in
+[Pull requests](https://github.com/open-telemetry/opentelemetry.io/pulls).
+
+After opening a PR, GitHub runs automated tests and tries to deploy a preview
+using [Netlify](https://www.netlify.com/).
+
+- If the Netlify build fails, select **Details** for more information.
+- If the Netlify build succeeds, select **Details** opens a staged version of
+  the OpenTelemetry website with your changes applied. This is how reviewers
+  check your changes.
+
+GitHub also automatically assigns labels to a PR to help reviewers.
+
+#### Changes from reviewers
+
+Sometimes reviewers commit to your pull request. Before making any other
+changes, fetch those commits.
+
+1. Fetch commits from your remote fork and rebase your working branch:
+
+   ```shell
+   git fetch origin
+   git rebase origin/<your-branch-name>
+   ```
+
+1. After rebasing, force-push new changes to your fork:
+
+   ```shell
+   git push --force-with-lease origin <your-branch-name>
+   ```
+
+#### Merge conflicts and rebasing
+
+If another contributor commits changes to the same file in another PR, it can
+create a merge conflict. You must resolve all merge conflicts in your PR.
```

Comment:
There is also an option to fix merge conflicts via the GitHub UI, this might be worth calling out.